### PR TITLE
Fixed ObjectManagerDecoratorTest (closes #772)

### LINKED ISSUE #772: Invalid data provider in the test suite
As per https://github.com/doctrine/common/pull/765#discussion_r99428560 by @Majkl578, the test suite currently loads an invalid data provider:

```
1) Warning
The data provider specified for Doctrine\Tests\Common\Persistence\ObjectManagerDecoratorTest::testAllMethodCallsAreDelegatedToTheWrappedInstance is invalid.
```

See Travis: https://travis-ci.org/doctrine/common/jobs/197349731#L228

### PR DESCRIPTION
**Pull request by ondrejmirtes**
Sorry for the mistake, I didn't notice that the test does not run :/

### CODE REVIEW

**Comment by Ocramius** [comment]
@ondrejmirtes shit happens, no need to be sorry :-)

File: tests/Doctrine/Tests/Common/Persistence/ObjectManagerDecoratorTest.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function getMethodParameters()
      */
     public function testAllMethodCallsAreDelegatedToTheWrappedInstance($method, array $parameters, $isVoidMethod)
     {
-        $returnedValue = $isVoidMethod ? 'INNER VALUE FROM ' . $method : null;
+        $returnedValue = !$isVoidMethod ? 'INNER VALUE FROM ' . $method : null;
```
Comment (Ocramius) [inline tests/Doctrine/Tests/Common/Persistence/ObjectManagerDecoratorTest.php]: Please just invert the ternary - adding a boolean negation just makes it more confusing

Comment (ondrejmirtes) [inline tests/Doctrine/Tests/Common/Persistence/ObjectManagerDecoratorTest.php]: @Ocramius Done!

**Comment by Ocramius** [comment]
👍

**Comment by Ocramius** [comment]
Linking #722

**Pull request merged**